arbitrary-file-write/file-delete/README.md

```diff
@@ -8,4 +8,14 @@
 * `rm /etc/group`
 	* system broken. tested: `sudo, login, su`
 * `rm /etc/sudoers`
-	* only `sudo` broken (also tested empty file)
\ No newline at end of file
+	* only `sudo` broken (also tested empty file)
+* `rm /etc/fstab`
+    * say goodbye to your system ig. most likely it'll boot to some rescue mode where you'll have to log in over a display or serial. 
```

Comment:
"most likely" is not confirmed. We should test this or be more clear about this being unconfirmed.